src/Visitor/Php/Symfony/FormTypeChoices.php

```diff
@@ -75,21 +75,31 @@ public function enterNode(Node $node)
             if (!$item->key instanceof Node\Scalar\String_) {
                 continue;
             }
+            $domain = null;
 
             if ('choices_as_values' === $item->key->value) {
                 $useKey = true;
 
                 continue;
             }
 
+            if ('choice_translation_domain' === $item->key->value) {
+                if ($item->value instanceof Node\Scalar\String_) {
+                    $domain = $item->value->value;
+                } elseif ($item->value instanceof Node\Expr\ConstFetch && $item->value->name->toString() === 'false') {
+                    $domain = false;
+                }
+            }
```

Comment:
I guess adding continue here would make sense.

Reply:
You are correct. Thanks!